frontend/src/component/common/Limit/Limit.tsx

```diff
@@ -10,7 +10,7 @@ import { ConditionallyRender } from '../ConditionallyRender/ConditionallyRender'
 const StyledBox = styled(Box)(({ theme }) => ({
     display: 'flex',
     flexDirection: 'column',
-    border: `2px solid ${theme.palette.background.application}`,
+    border: `1px solid ${theme.palette.divider}`,
```

Comment:
adjusting color of the limit component borders so that it looks good in dark mode and is aligned with other lines in the strategy ui